axlearn/common/flash_attention/tpu_attention_test.py

```diff
@@ -49,6 +50,19 @@ class TestFlashAttention(TestCase):
         ),
     ]
 
+    @parameterized.product(seq_len=[8, 16, 32, 128], sliding_window_size=[4, 8, 16])
+    def test_sliding_window_mask(self, seq_len, sliding_window_size):
+        shape = (seq_len, seq_len)
+        ref_mask = splash_attention_mask.LocalMask(
+            shape=shape, window_size=(sliding_window_size, None), offset=0
```

Comment:
According to the documentation of splash_attention_mask.LocalMask(), setting window_size=(sliding_window_size, None) would make the mask unbounded on the right, which would not be the same behavior as sliding_window_causal_mask().

Reply:
It is fixed in https://github.com/apple/axlearn/pull/758

Reply:
Thanks for fixing!